utils/scripts/install-centos8.sh

```diff
@@ -10,6 +10,8 @@
 
 set -e
 
+fedora_java=$1
```

Comment:
This isn't really anything to do with _Fedora_ is it?  It's just whether to install dependencies for the Java client build, right?  Even though currently we only do that on Fedora, it could be done on other platforms also, right?

```suggestion
java=$1
```

Reply:
yes. it's really about fedora. Java-11 is installed along with maven automatically if we install maven from repo in fedora. I tried several ways to remove the Java-11 dep but failed. Thus, I need to handle fedora separately. 

Since you mentioned no Java and maven at all if it builds without Java. I split "fedora_java" to two variables, "java" and "fedora".